packages/cli/internal/pkg/cli/context/info.go

```diff
@@ -18,6 +22,18 @@ func (i Summary) IsEmpty() bool {
 	return reflect.ValueOf(i).IsZero()
 }
 
+//IsServerProcessEngine Does the workflow engine run as a server process? A server process has a potential one to many
+// mapping with workflow runs.
+func (i *Summary) IsServerProcessEngine() bool {
```

Comment:
`i` is usually used as an index in loops. I would recommend to rename it to `s`.